ckb-vm-syscall-tracer/protos/traces.proto

```diff
@@ -8,6 +8,10 @@ message IoData {
   uint64 additional_length = 2;
 }
 
+message IoDataAsCode {
```

Comment:
Should we use a new type, or should we reuse `IoData`(but restricting `additional_length` to 0?

Personally I would just reuse `IoData`

Reply:
Of course, reusing the previous data structure is more concise, I will modify it.

Reply:
I thought about it and it's probably not possible, because you designed IoData to handle [Partial Loading](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0009-vm-syscalls/0009-vm-syscalls.md#partial-loading), but [Load Cell Data As Code](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0009-vm-syscalls/0009-vm-syscalls.md#load-cell-data-as-code) is not designed with partial loading in mind. If I choose to share IoData, I won't be able to distinguish them in [apply_partial_content](https://github.com/nervosnetwork/ckb-vm-contrib/blob/main/ckb-vm-syscall-tracer/src/lib.rs#L648-L665).

Reply:
I know what to do. For the two questions above, I think I can:

1. No.
2. Yes.